README.md

```diff
@@ -1,9 +1,117 @@
 # OpenPubkey Reference Implementation
 
-OpenPubkey adds user generated cryptographic signatures to OpenID Connect (OIDC) to enable users to sign messages or artifacts under their OpenID identity. Verifiers can check that these signatures are valid and associated with the signing OpenID identity. OpenPubkey does not add any new trusted parties beyond what is required for OpenID Connect and is fully compatible with existing OpenID Providers (Google, Azure/Microsoft, Okta, OneLogin, Keycloak) without any changes to the OpenID Provider.
-
 This repo contains the current reference implementation of OpenPubkey. The reference implementation is a work in progress.
 
+## What is OpenPubkey?
+
+OpenPubkey adds user or workload generated public keys to OpenID Connect (OIDC) enabling identities to sign messages or artifacts under their OIDC identity. In essence, OpenPubkey is a protocol for getting OpenID Providers (OPs) to bind identities to public keys.
+
+Verifiers can check that these signatures are valid and associated with the signing OpenID identity. OpenPubkey does not add any new trusted parties beyond what is required for OpenID Connect. It is fully compatible with existing OpenID Providers (Google, Azure/Microsoft, Okta, OneLogin, Keycloak) without any changes to the OpenID Provider.
+
+### ID Tokens and PK Tokens
+
+OpenPubkey uses ID Tokens issued by OpenID Providers (OPs) to produce PK Tokens. A PK Token consist of the ID Token and the CIC (Client-Instance Claims) where the CIC contains the user's public key and associated metadata. The ID Token contains a hash of the CIC, so that the values in the CIC including the identities public key are cryptographically bound to the ID Token signed by the OP.
```

Comment:
"identities" -> "identity's"

Reply:
@EthanHeilman can this be fixed?

Reply:
@whalelines Vscode was telling me the file was saved with this being fixed and git diff was saying no differences. Took me about twenty minutes to figure this out, but github disagreeing. Turns out vscode let me open two windows to same file, which triggered a bug in vscode so that it got confused about the actual content of the file. Had to close vscode to fix it. I was starting to question my sanity

Should be fixed now.